scss/bootstrap/overrides/style/_ui-bootstrap.scss

```diff
@@ -11,7 +11,6 @@
             }
 
             tbody {
-                background-color: $crm-brand-primary-light;
```

Comment:
we don't need this anymore?

Reply:
This is not affecting component design. it's tested